.vscode/launch.json

```diff
@@ -27,7 +27,11 @@
             ],
             "outFiles": [
                 "${workspaceFolder}/out/test/**/*.js"
-            ]
+            ],
+            "skipFiles": [
+                "${workspaceFolder}/node_modules/**/*.js",
```

Comment:
why are we adding this?

Reply:
Just don't want to debug into so many external files during unit test debug.